support-frontend/app/utils/CheckoutValidationRules.scala

```diff
@@ -155,11 +163,23 @@ object DigitalPackValidation {
         currencyIsSupportedForCountry(country, currency) and
         PaidProductValidation.noEmptyPaymentFields(paymentFields)
 
+    def isInBenefitsTest(supportAbTests: Set[AbTest]) = {
+      if (
+        isValidBenefitsTestPurchase(
+          createSupportWorkersRequest.product.asInstanceOf[DigitalPack],
```

Comment:
rather than casting at run time, just change this line to pass in the product as a `DigitalPack`
https://github.com/guardian/support-frontend/blob/b13ecceb880dbf292b22f87524f014292732d991/support-frontend/app/utils/CheckoutValidationRules.scala#L36